pkgs/stdenv/cross/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
-, system, platform, crossSystem, config
+, localSystem, crossSystem, config
```

Comment:
I'll expand https://github.com/NixOS/nixpkgs/pull/21268/files#diff-69df24a05ec3d577cb638113e9837ae1R86. The searching ones can go in https://github.com/NixOS/nixpkgs/pull/21268/files#diff-017a38a631b06991d33857f7681874b3R83.

Oh, and the manual should duplicate this more or less.